### DISCUSSION TITLE
Use Treeview Prism plugin?

### DISCUSSION THREAD

**Discussion by sharky98** [facebook/docusaurus#9753] [Q&A] [answered]
Hello,

I would like to use the [Treeview](https://prismjs.com/plugins/treeview/) plugin for Prism. But I can't seems to find any places to tell docusaurus to load that plugin?

Thanks

**Comment by slorber** [comment, marked as answer]
We don't support Prism plugins because the lib we use in our theme "react-prism-renderer" does not support Prism plugins.

See:
- https://github.com/FormidableLabs/prism-react-renderer/issues/2
- https://github.com/FormidableLabs/prism-react-renderer/issues/90
- https://github.com/FormidableLabs/prism-react-renderer/issues/154
- https://github.com/facebook/docusaurus/issues/3318

---

However, the good news is that you can implement your own code block component, or enhance our component thanks to [Swizzling](https://docusaurus.io/docs/swizzling) and implement this feature yourself if you want. It's possible that you use a totally different rendering logic for treeviews that doesn't use react-prism-renderer.
